### ISSUE TITLE
[Adaptive Cards> FoodOrder.json]: [Adaptive Cards> FoodOrder.json]: Options under 'Steak' are not accessible through the swipe gesture under 'Food Order' screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7585]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.3.1-beta.20220620.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 15.5
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.20220612.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'FoodOrder.json' and activate it.
7. Navigate to the 'Steak' option and double tap to activate it.
8. Perform right swipe and observe the issue.

**Actual Result:**
After activating the 'Steak' button options under 'Steak' are not accessible through the swipe gesture.

**Expected Result:**
Options under 'Steak' options should be accessible through the swipe gesture.

**Note:**
Same issue is observed with 'Tofu' and 'Chicken' buttons as well.

**User Experience:**
It will impact the screen reader user if the items are not accessible through the swipe gesture as they are not able to perform desired action.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.01.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/176353179-e819df27-41ee-46fa-aff3-3a89e7f6a04d.MP4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro on the below build hence closing this issue.

Build Version -1.0(2.4.0-beta.20220718.1)
